langwatch/prisma/migrations/20260802120001_webhook_endpoints_platform/migration.sql

```
@@ -0,0 +1,56 @@
+    "latencyMs" INTEGER,
+    "error" TEXT,
+    "response" JSONB,
+    "outcome" "WebhookDeliveryOutcome" NOT NULL,
+    "firedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "WebhookEndpointDelivery_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE INDEX "WebhookEndpoint_organizationId_status_idx" ON "WebhookEndpoint"("organizationId", "status");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)